### PR TITLE
Use the Konflux logo in the UI

### PR DESCRIPTION
**Pull request by gbenhaim**
Mount the Konflux logo over the existing logo.

image

### CODE REVIEW

**Review by yftacherzog — APPROVED** [review]

**Review by avi-biton — APPROVED** [review]
lgtm

File: konflux-ci/ui/core/chrome/chrome-frontend.yaml [hunk 1]
```diff
@@ -79,6 +79,10 @@ spec:
               name: config
             - mountPath: /opt/app-root/src/build/preview/operator-generated
               name: config
+            - mountPath: /opt/app-root/src/build/stable/js/d6083bde9a964ab4.svg
+              subPath: d6083bde9a964ab4.svg
+              name: logo
+              readOnly: true
```
Comment (eisraeli) [inline konflux-ci/ui/core/chrome/chrome-frontend.yaml:85]: Is this line needed?
Doesn't `readOnlyRootFilesystem: true` under `securityContext` specifies that the entire root filesystem of the container should be mounted as read-only ?

File: konflux-ci/ui/core/chrome/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+The Konflux logo is mounted using a configmap.
+After changing the chrome image, ensure that the Konflux logo is still visible in the UI,
+if not, you may need to change the mount path of the logo.
```
Comment (eisraeli) [inline konflux-ci/ui/core/chrome/README.md]:
```suggestion
After changing the chrome image, ensure that the Konflux logo is still visible in the UI; If not, you may need to change the mount path of the logo.
```

File: konflux-ci/ui/core/chrome/chrome-frontend.yaml (hunk 1 above)
Comment (gbenhaim) [reply konflux-ci/ui/core/chrome/chrome-frontend.yaml:85]: I'm pretty sure that `readOnlyRootFilesystem: true` applies only to the files bundled in the container image, not mounted files.

**Review by eisraeli — APPROVED** [review]

**Pull request merged**